test_pkgconfig.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import pytest
+
+import pkgconfig
```

Comment:
Could you remove the newlines between the imports?

Reply:
Sure — removed.

Reply:
Thank :+1: